Chamasma/Presentation/Sources/Presentation/Examples/Meal+Examples.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Menu+Examples.swift
+//  
+//
+//  Created by Mariam Ormotsadze on 23.04.23.
+//
+
+import Domain
+import DesignSystem
+
+// MARK: - Example
+public extension Meal {
+    enum examples: CaseIterable {
+        case cheesburger
+        case chiliCheesburger
+        case hamburger
+        case vegetarianBurger
+        case tofu
+        case salad
```

Comment:
instead of creating cases, we can have `static let cheesburger = Meal(...)` and so on codes